extensions/wikia/CityVisualization/models/CityVisualization.class.php

```diff
@@ -588,19 +588,19 @@ protected function getWikiImagesConditions($wikiId, $langCode, $filter) {
 		$conditions ['city_lang_code'] = $langCode;
 
 		switch ($filter) {
-			case ImageReviewStatuses::STATE_APPROVED:
-				$conditions ['image_review_status'] = ImageReviewStatuses::STATE_APPROVED;
+			case ImageStates::APPROVED:
+				$conditions ['image_review_status'] = ImageStates::APPROVED;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "image_review_status" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)